### PR TITLE
updates for release of beta4

### PR DESCRIPTION
**Pull request by thomaszurkan-optimizely**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # Optimizely Objective-C SDK Changelog
+## 2.0.2-beta4
+August 1, 2018
+
+** This is beta 4 and a release candidate.  There are several things to note about this pre-release.  This release includes Feature Management and is backward compatible. The APIs mentioned in beta 3 are included.  
+
+### Bug Fixes:
+* Force builderWithBlock for OPTLYManagerBuilder
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: nit. Period at the end.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+* Return nil for getFeatureVariable[Integer,Double,Boolean] if the value type is incorrect or the feature variable does not exist.
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: String as well right?

**Review by aliabbasrizvi — APPROVED** [review]
Looks good. Minor nits.

File: CHANGELOG.md (hunk 2 above)
Comment (thomaszurkan-optimizely) [reply CHANGELOG.md]: String was already working correctly.

**Pull request merged**